### PR TITLE
Bump chainlink-solana

### PR DESCRIPTION
**Pull request by DylanTinianov**
Bump chainlink-solana and fix tests which broke due to new TOML configuration being added for Solana MultiNode.
Ticket: https://smartcontract-it.atlassian.net/browse/BCFR-960

### Resolves

https://smartcontract-it.atlassian.net/browse/BCFR-960

### CODE REVIEW

File: core/services/chainlink/config_test.go [hunk 1]
```diff
@@ -1026,7 +1089,7 @@ BlockBackfillDepth = 100
 BlockBackfillSkip = true
 ChainType = 'Optimism'
 FinalityDepth = 42
-FinalityTagEnabled = true
+FinalityTagEnabled = false
```
Comment (dhaidashenko) [inline core/services/chainlink/config_test.go]: why this change?

File: core/services/chainlink/config_test.go [hunk 2]
```diff
@@ -1103,7 +1166,7 @@ MaxBufferSize = 17
 SamplingInterval = '1h0m0s'
 MaxAllowedFinalityDepth = 1500
 FinalityTagBypass = false
-PersistenceEnabled = false
+PersistenceEnabled = true
```
Comment (dhaidashenko) [inline core/services/chainlink/config_test.go]: why this change?

File: core/services/chainlink/config_test.go [hunk 3]
```diff
@@ -710,12 +754,31 @@ func TestConfig_Marshal(t *testing.T) {
...
+					FinalityDepth:                ptr[uint32](0),
+					FinalityTagEnabled:           ptr(true),
+					FinalizedBlockOffset:         ptr[uint32](0),
+				},
 			},
 			Nodes: []*solcfg.Node{
 				{Name: ptr("primary"), URL: commoncfg.MustParseURL("http://solana.web")},
-				{Name: ptr("foo"), URL: commoncfg.MustParseURL("http://solana.foo"), SendOnly: true},
-				{Name: ptr("bar"), URL: commoncfg.MustParseURL("http://solana.bar"), SendOnly: true},
+				{Name: ptr("foo"), URL: commoncfg.MustParseURL("http://solana.foo")},
```
Comment (dhaidashenko) [inline core/services/chainlink/config_test.go]: Why this change?

**Review by aalu1418 — APPROVED** [review]

File: core/config/docs/chains-solana.toml [hunk 4]
```diff
@@ -39,6 +39,38 @@ BlockHistoryPollPeriod = '5s' # Default
...
+PollFailureThreshold = 5 # Default
+# PollInterval is the rate to poll for node health.
+PollInterval = '10s' # Default
+# SelectionMode is the method used to select the next best node to use.
+SelectionMode = 'PriorityLevel' # Default
+# SyncThreshold is the number of blocks behind the best node that a node can be before it is considered out of sync.
+SyncThreshold = 5 # Default
+# NodeIsSyncingEnabled enables the feature to avoid sending transactions to nodes that are syncing. Not relavant for Solana.
+NodeIsSyncingEnabled = false # Default
+# LeaseDuration is the max duration a node can be leased for.
```
Comment (jmank88) [inline core/config/docs/chains-solana.toml:55]: Can we please do another pass on these docs? Phrases like this are skirting the documentation requirement by saying nothing in particular, and in this case we already have good EVM examples to draw on.

**Review by jmank88 — CHANGES_REQUESTED** [review]

**Review by jmank88 — APPROVED** [review]

File: core/config/docs/chains-solana.toml (hunk 4 above)
Comment (jmank88) [reply core/config/docs/chains-solana.toml:55]: I merged to unblock other PRs, but please circle back to this.

Comment (DylanTinianov) [reply core/config/docs/chains-solana.toml:55]: Yeah sounds good, I'll create a ticket for later!
